src/main/resources/logback-spring.xml

```diff
@@ -3,59 +3,34 @@
     <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
     <property name="LOG_PATTERN"
               value="%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m | %mdc %n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}"/>
-
     <springProperty scope="context" name="app" source="spring.application.name"/>
-
-
-    <springProfile name="dev || stdout">
-        <appender name="consoleAppender" class="ch.qos.logback.core.helpers.NOPAppender"/>
-        <appender name="logAppender" class="ch.qos.logback.core.ConsoleAppender">
-            <layout class="ch.qos.logback.classic.PatternLayout">
-                <Pattern>${LOG_PATTERN}</Pattern>
-            </layout>
-        </appender>
-    </springProfile>
-
-    <springProfile name="!(dev || stdout)">
-        <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
-            <layout class="ch.qos.logback.classic.PatternLayout">
-                <Pattern>${LOG_PATTERN}</Pattern>
-            </layout>
-        </appender>
-        <appender name="logAppender" class="com.microsoft.applicationinsights.logback.ApplicationInsightsAppender"/>
-    </springProfile>
-
+    
+    <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+        <layout class="ch.qos.logback.classic.PatternLayout">
+            <Pattern>${LOG_PATTERN}</Pattern>
+        </layout>
+    </appender>
+    
     <logger name="uk.gov.justice.digital.hmpps.courthearingeventreceiver.CourtHearingEventReceiverKt" additivity="false"
             level="DEBUG">
-        <appender-ref ref="logAppender"/>
         <appender-ref ref="consoleAppender"/>
     </logger>
-
     <logger name="uk.gov.justice.digital.hmpps" additivity="false" level="DEBUG">
-        <appender-ref ref="logAppender"/>
         <appender-ref ref="consoleAppender"/>
     </logger>
-
     <logger name="org.springframework" additivity="false" level="INFO">
-        <appender-ref ref="logAppender"/>
+        <appender-ref ref="consoleAppender"/>
     </logger>
-
     <logger name="com.microsoft.applicationinsights" additivity="false" level="INFO">
```

Comment:
Should we have the app insights connection string in the secrets [more info](https://learn.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-upgrade-from-2x#step-3-configure-your-application-insights-connection-string). I don't know if the instrumentation key is enough 

I think logback is autoinstrumented for application insights so the section `<logger name="com.microsoft.applicationinsights"`  can be removed